docs/rbe/rbe.md

```diff
@@ -0,0 +1,161 @@
+# RBE for Flutter Engine Developers
+
+## Overview
+
+This is documentation on setting up RBE for building the Flutter engine. It is
+expected to work for Googlers only on corp Linux, macOS, and Windows hosts,
+including cloudtop instances.
+
+## Getting started
+
+The first step is to add an entry to the `.gclient` file. The entry to add is
+`"use_rbe": True` in the `custom_vars` section. It should look like this:
+
+After making this edit, you must be authenticated as a Googler by CIPD so that
+the RBE configurations can be downloaded from the `flutter_internal`
+[CIPD bucket](https://chrome-infra-packages.appspot.com/p/flutter_internal):
+
+After authentication successfully, run `gclient sync -D`.
+
+## Running an RBE build
+
+In the engine repo, all RBE builds must be initiated through the `et` tool whose
+entrypoint is the script `//flutter/bin/et`. This is so that the local RBE
+proxy is correctly initialized and shut down around invocations of `ninja`.
+
+### gcloud
+
+Before running an RBE build, you must be authenticated with the Google cloud
+project that owns the RBE worker pool. A gcloud SDK is checked out with the
+engine repo under e.g. `//flutter/buildtools/mac-arm64/gcloud`.
+(Replace `mac-arm64` with the directory name that is correct for your host
+platform.)
+
+The `gcloud` tool in this SDK must be on your path. The tool lives under
+`//flutter/buildtools/mac-arm64/gcloud/bin`, which is the path to add to your
+`PATH` environment variable. Alternatively, you can get the gcloud SDK on your
+path by installing it on your system by following the instructions at
+[https://cloud.google.com/sdk/docs/install]
+(https://cloud.google.com/sdk/docs/install).
```

Comment:
I think this line break is preventing the link from working properly, but also I think it would still show up as a link without the markdown formatting.